.drone.yml

```diff
@@ -1,3 +1,13 @@
+# ==============================================================================
```

Comment:
Why these changes belong to this issue?

Reply:
Also there is a ticket for drone cache issue which will be a good candidate for this issue.
[Jira Ticket](https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-2288)

Reply:
These changes were pushed into this issue to see if the build would be successfull.
I will create a new PR with those changes only now.

Reply:
fixed in #292